app/concerns/cmr_collections_helper.rb

```diff
@@ -19,7 +19,11 @@ def get_revisions(concept_id, revision_id)
       break if latest && latest['meta']['revision-id'] >= revision_id.to_i
       attempts += 1
       sleep 0.05
+      Rails.logger.info("Retrieving a collection record in `get_revisions`. Need to loop, about to try attempt number #{attempts}")
```

Comment:
This is very minor semantics but when I read this I thought it might sound better as "make attempt" rather than "try attempt"